### ISSUE TITLE
Webpack Error with `Yarn RW Dev`

### ISSUE THREAD

**Issue by ngoctranfire** [redwoodjs/redwood#5753]
I can't seem to get yarn Rw Dev. This is after updating my Mac to the latest version. I'm on the 2019 Intel Mac Version, 16 inch.
I'm currently on Mac OS Monterey, Version 12.4.

Would greatly appreciate any help in trying to debug this and fixing this issue. I can't do yarn rw dev at all.

```
gen | Generating TypeScript definitions and GraphQL schemas...
gen | 14 files generated
web | [webpack-cli] TypeError: cli.isMultipleCompiler is not a function
web |     at Command.<anonymous> (.../supabase-redwoodjs-tutorial/node_modules/@webpack-cli/serve/lib/index.js:146:35)
web |     at async Promise.all (index 1)
web |     at async Command.<anonymous> (../supabase-redwoodjs-tutorial/node_modules/webpack-cli/lib/webpack-cli.js:1674:7)
web | cd ".../supabase-redwoodjs-tutorial/web" && yarn cross-env NODE_ENV=development RWJS_WATCH_NODE_MODULES= webpack serve --config ".../supabase-redwoodjs-tutorial/node_modules/@redwoodjs/core/config/webpack.development.js"  exited with code 1
```

**Comment by The-Bot11** [comment]
Must update webpack-cli to 4.10.0 

how i fix this issue : 

- installed the latest version of webpack-cli@4.10.0 & webpack@4.73.0
- remove webpack-cli & webpack folders from "/node_modules/@redwoodjs/core/node_modules/"

that work for me

**Comment by The-Bot11** [comment]
Hello, I have same issue

it just need update webpack-cli to 4.10.0

how i fix this issue :

-     installed the latest version of webpack-cli@4.10.0 & webpack 
-     remove webpack-cli & webpack folders from "/node_modules/@redwoodjs/core/node_modules/"

**Comment by The-Bot11** [comment]
Hello, I have same issue

it just need update webpack-cli to 4.10.0

how i fix this issue :

-  installed the latest version of webpack-cli@4.10.0 & webpack
- remove webpack-cli & webpack folders from "/node_modules/@redwoodjs/core/node_modules/"

**Comment by jtoar** [comment]
Thanks for the report @ngoctranfire, according to this issue in webpack-cli, upgrading to v4.10 should fix it. We've got a PR in progress now:
- https://github.com/webpack/webpack-cli/issues/3294
- https://github.com/redwoodjs/redwood/pull/5756

**Comment by jtoar** [comment]
This has been patched in v1.5.2: https://github.com/redwoodjs/redwood/releases/tag/v1.5.2. Please upgrade via `yarn rw upgrade` and let us know how it goes!
